README.md

```diff
@@ -1,5 +1,5 @@
 # React article
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_article/)
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://VVHimiak.github.io/react_article/)
```

Comment:
Your demo link in conversation doesn't work, write the correct link
http://joxi.ru/n2YPpW4c7dnMom

Reply:
https://user-images.githubusercontent.com/29519969/131699712-997241e4-f531-4b12-9645-4911ee2a5803.png